### PR TITLE
Front page school layer

### PR DESCRIPTION
**Pull request by usr110**
hi,
I have created a new HTML page which shows both school travel and commute layer data. 

image

Dealing with JS is quite a pain. Had to re-read what I'd done for [la-map](https://pct.bike/la-map.html). Now there are three outstanding issues - two major and one minor.  @nikolai-b your inputs would be really useful here.

- [x] For some reason the default control is not triggering load of data. So when the map loads, you don't see anything. You need to switch `commute` to `school`, to load the data. I had that issue in `la-map`. I followed [this](https://stackoverflow.com/a/33762133). See lines: https://github.com/npct/pct-shiny/blob/0faef5287b042c0c47c962cbe30d7647c9c0b7f1/regions_www/la-map.html#L592-L596

- [x]  Once it's working fine, replace the embedded `map.js` with `test.html`. This means loading both `commute` and `school` geojsons file on document load. 

- [x]  Minor issue: as @AnnaGoodman1 mentioned that the `school-layer` is missing `Wales`. We can either leave it out or put `NA` there. This needs to be done.

### CODE REVIEW

**Comment by AnnaGoodman1** [comment]
Thanks. re. WAles, either could be fine - leave it out if possible, otherwise NA

**Comment by Robinlovelace** [comment]
Looking good @nikolai-b, many thanks for picking this up.

File: regions_www/test.html [hunk 1]
```diff
@@ -397,6 +392,7 @@
   };
 
   legend.addTo(map);
+  map.whenReady(layerUpdate);
```
Comment (usr110) [inline regions_www/test.html]: Brilliant, thanks @nikolai-b !

**Comment by usr110** [comment]
> 
> 
> Thanks. re. WAles, it needs to be there for commuting - so I would vote leave it outfor the schools layer only if possible, otherwisekeep it in and mark it NA

Okay in that sense, it's already dealt with. `Wales` does not appear in the school-layer. Thanks @AnnaGoodman1

**Comment by nikolai-b** [comment]
The map looks really good :clap:, great work @usr110 .  I just did a few changes during nap time and I plan to do a few more now.  I'll make a pull request next time so you @usr110 can review.

**Comment by usr110** [comment]
> 
> 
> The map looks really good clap, great work @usr110 . I just did a few changes during nap time and I plan to do a few more now. I'll make a pull request next time so you @usr110 can review.

Thanks a bunch, @nikolai-b. Great work there! :+1:

**Comment by usr110** [comment]
@nikolai-b has so graciously fixed all the bugs. If @Robinlovelace you have the time - and energy, to test it, then please go ahead. From my side, it's ready to be merged.

Heads up @AnnaGoodman1

**Comment by nikolai-b** [comment]
@usr110 as this only pushes to the test server I'm merging so they can test there.

:tada:

**Comment by usr110** [comment]
Fine and thanks @nikolai-b 

Opps..something has gone wrong. The map's empty - haven't found the GeoJsons, it seems.

**Comment by usr110** [comment]
Fixed in: https://github.com/npct/pct-shiny/commit/ef1401181390f1c5eee16422fe7d4d76327da8ec

@nikolai-b

**Pull request merged**